packages/flutter_tools/lib/src/commands/build_apk.dart

```diff
@@ -333,7 +322,8 @@ int _buildApk(
     File apkShaFile = new File('$outputFile.sha1');
     apkShaFile.writeAsStringSync(calculateSha(finalApk));
 
-    printStatus('Generated APK to ${finalApk.path}.');
+    double size = finalApk.lengthSync() / (1024 * 1024);
```

Comment:
Kinda lame. I love Dart's Duration class, I'm slightly surprised they don't have something similar for file sizes. :D